### PR TITLE
Grammar fix

### PR DESCRIPTION
**Pull request by AYS17**
fixing the "Assigned to null" things.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/csharp/nullable-references.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ C# 8.0 introduces **nullable reference types** and **non-nullable reference type
 
 This new feature provides significant benefits over the handling of reference variables in earlier versions of C# where the design intent can't be determined from the variable declaration. The compiler didn't provide safety against null reference exceptions for reference types:
 
-- **A reference can be null**. The compiler doesn't issue warnings when a reference type is initialized to null, or later assigned to null. The compiler issues warnings when these variables are dereferenced without null checks.
+- **A reference can be null**. The compiler doesn't issue warnings when a reference type is initialized to null, or later assigned null. The compiler issues warnings when these variables are dereferenced without null checks.
```
Comment (pkulikov) [inline docs/csharp/nullable-references.md:20]:
```suggestion
- **A reference can be null**. The compiler doesn't issue warnings when a reference-type variable is initialized to `null`, or later assigned `null`. The compiler issues warnings when these variables are dereferenced without null checks.
```

File: docs/csharp/nullable-references.md [hunk 2]
```diff
@@ -110,10 +110,10 @@ In an enabled nullable annotation context, the `?` character appended to a refer
...
 
-1. The variable has been definitely assigned to a non-null value.
+1. The variable has been definitely assigned a non-null value.
 1. The variable or expression has been checked against null before de-referencing it.
 
-The compiler generates warnings when you dereference a variable or expression that is **maybe null** in a nullable warning context. Furthermore, the compiler generates warnings when a nonnullable reference type is assigned to a **maybe null** variable or expression in an enabled nullable annotation context.
+The compiler generates warnings when you dereference a variable or expression that is **maybe null** in a nullable warning context. Furthermore, the compiler generates warnings when a nonnullable reference type is assigned a **maybe null** variable or expression in an enabled nullable annotation context.
```
Comment (pkulikov) [inline docs/csharp/nullable-references.md:116]:
```suggestion
The compiler generates warnings when you dereference a variable or expression that is **maybe null** in a nullable warning context. Furthermore, the compiler generates warnings when a nonnullable reference-type variable is assigned a **maybe null** variable or expression in an enabled nullable annotation context.
```

**Review by pkulikov — COMMENTED** [review]
Further tweaks on top of the added ones.

File: docs/csharp/nullable-references.md (hunk 1 above)
Comment (Forgind) [inline docs/csharp/nullable-references.md:20]:
```suggestion
- **A reference can be null**. The compiler doesn't issue warnings when a reference type is initialized to null or later assigned null. The compiler issues warnings when these variables are dereferenced without null checks.
```

**Review by IEvangelist — APPROVED** [review]

**Pull request merged**